### DISCUSSION TITLE
启动FE节点异常

### DISCUSSION THREAD

**Discussion by liuzhuo19940206** [apache/doris#22119] [A - General / Q&A]
<img width="1082" alt="image" src="https://github.com/apache/doris/assets/22168565/b84738df-f9d6-4cb0-b7b4-1ba596a02754">

**Comment by liuzhuo19940206** [comment]
环境是Mac系统，doris的版本是：1.2.6，安装的是二进制包

**Comment by LemonLiTree** [comment]
Did you download this package?
[apache-doris-1.2.6-bin-arm.tar.xz](https://apache-doris-releases.oss-accelerate.aliyuncs.com/apache-doris-1.2.6-bin-arm64.tar.xz)
